packages/gunshi/src/types.ts

```
@@ -156,18 +160,21 @@ export interface CommandEnvironment<G extends GunshiParamsConstraint = DefaultGu
    * Hook that runs before any command execution
    * @see {@link CliOptions.onBeforeCommand}
+   * @since v0.24.0
    */
   onBeforeCommand: ((ctx: Readonly<CommandContext<G>>) => Awaitable<void>) | undefined
   /**
    * Hook that runs after successful command execution
    * @see {@link CliOptions.onAfterCommand}
+   * @since v0.24.0
    */
   onAfterCommand:
     | ((ctx: Readonly<CommandContext<G>>, result: string | void) => Awaitable<void>)
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)